### PR TITLE
akku: init at 1.1.0

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change

Add https://akkuscm.org/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 136096` at 313e62b7 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>akku</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/tools/package-management/akku/default.nix:13:5:
```
   |
13 |     (substituteAll {
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/tools/package-management/akku/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchurl, autoreconfHook, pkg-config, guile, curl, substituteAll }:
+
+stdenv.mkDerivation rec {
+  pname = "akku";
+  version = "1.1.0";
+
+  src = fetchurl {
```
Comment (figsoda) [inline pkgs/tools/package-management/akku/default.nix:7]: use `fetchFromGitLab`

File: pkgs/tools/package-management/akku/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    url = "https://gitlab.com/akkuscm/akku/-/archive/v${version}/akku-v${version}.tar.gz";
+    sha256 = "1ddxg2vk7rl9a7mqix05qxpvric5p5rl6w4qcm8abvgg39w2nbjp";
+  };
+
+  patches = [
+    (substituteAll {
```
Comment (figsoda) [inline pkgs/tools/package-management/akku/default.nix:13]: would be nice to have a patch comment

File: pkgs/tools/package-management/akku/default.nix [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      src = ./hardcode-libcurl.patch;
+      libcurl = "${curl.out}/lib/libcurl${stdenv.hostPlatform.extensions.sharedLibrary}";
+    })
+  ];
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  buildInputs = [ guile ];
```
Comment (figsoda) [inline pkgs/tools/package-management/akku/default.nix:21]: are you sure guile is not in  `nativeBuildInputs`?

File: pkgs/tools/package-management/akku/default.nix [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+  makeFlags = [ "GUILE_AUTO_COMPILE=0" ];
+
+  meta = with lib; {
+    homepage = "https://akkuscm.org/";
+    description = "Language package manager for Scheme";
+    changelog = "https://gitlab.com/akkuscm/akku/-/raw/v${version}/NEWS.md";
+    platforms = platforms.all;
+    license = licenses.gpl3Plus;
+    maintainers = [ maintainers.marsam ];
```
Comment (figsoda) [inline pkgs/tools/package-management/akku/default.nix:36]:
```suggestion
    maintainers = with maintainers; [ marsam ];
```

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/tools/package-management/akku/default.nix (hunk 3 above)
Comment (marsam) [reply pkgs/tools/package-management/akku/default.nix:24]: yes, guile is required at runtime. Akku is a Scheme script.

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 136096` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>akku</li>
  </ul>
</details>

**Review by figsoda — APPROVED** [review]

**Pull request merged**